### DISCUSSION TITLE
Verify

### DISCUSSION THREAD

**Discussion by remadrivan** [community/community#40747] [Other Feature Feedback, Questions, & Ideas]
How to verify accounts?

**Comment by mohamedzhioua** [comment]
In the "Access" section of the sidebar, click Emails. Under your email address, click Resend verification email. GitHub will send you an email with a link in it. After you click that link, you'll be taken to your GitHub dashboard and see a confirmation banner.
or you can just take a look here in the below link and follow the steps : 
"https://docs.github.com/en/get-started/signing-up-for-github/verifying-your-email-address"

**Comment by ghostinhershell** [comment]
Welcome to the GitHub Community, @rexus3512, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
